### PR TITLE
Added example for array operators

### PR DESCRIPTION
**Pull request by deeptikorwar**
Addresses #7671 

@Amruta-Ranade, please review this PR and let me know if any changes are required. I have slightly modified the example specified in the issue to keep the other examples on the page intact.

### CODE REVIEW

**Comment by rytaft** [comment]
For users not familiar with `@>` and `<@`, it might help to explicitly state that `@>` means "contains" and  `<@` means "is contained by". 

Optionally, in addition to the examples you already included, you might consider also providing an example like `... WHERE d <@ ARRAY[10,20,30,50]`, which should return the one row in the table. @amruta-ranade may have an opinion about whether or not that's worthwhile.

**Comment by Amruta-Ranade** [comment]
+1 to explicitly stating meanings for new users.
As for the additional example: @ericharmeling, can you advise if you have the bandwidth?

**Comment by deeptikorwar** [comment]
@rytaft and @Amruta-Ranade, I’m planning to add the following line to explain `<@` and `@>`. Please let me know if this works.

You can use the following operators to run containment queries on array columns:
* "is contained by": `<@`
* "contains": `@>`

Re the extra example, @rytaft, do you mean `SELECT * FROM c WHERE d @> ARRAY[10,20,30];` as “50” is added to the array in a different example. Also, as both `SELECT * FROM c WHERE d @> ARRAY[10,20];` and `SELECT * FROM c WHERE d @> ARRAY[10,20,30];` return the same row, can we mention a scenario where users might want to choose one statement over the other?

**Comment by rytaft** [comment]
>  @rytaft, do you mean SELECT * FROM c WHERE d @> ARRAY[10,20,30];

No, I meant what I wrote above. The full query is: `SELECT * FROM c WHERE d <@ ARRAY[10,20,30,50];`. My example is different and may be helpful to readers because it shows the "is contained by" operator in action, in an example where the predicate is true. Your example with "is contained by", `SELECT * FROM c WHERE ARRAY[42,99] <@ d;`, is less helpful in my opinion because the variable is on the right hand side and `ARRAY[42,99]` doesn't overlap with the array in column `d` (`ARRAY[10,20,30]` at that point in the doc). I don't think it's bad to also show your example, but if we need to choose, I would personally prefer something like my example (feel free to tweak it and replace 50 with one or more other numbers).

Note that I wouldn't change your other example, `SELECT * FROM c WHERE d @> ARRAY[10,20];`. I think it's a good demonstration of "contains".

**Review by ericharmeling — COMMENTED** [review]
Thanks for your contribution @deeptikorwar !

As a general note, please make sure that you are using the correct version of CockroachDB when you paste output in examples (as of 10/20/20, https://www.cockroachlabs.com/docs/releases/v20.1.7.html and https://www.cockroachlabs.com/docs/releases/v20.2.0-rc.1.html). I believe 20.1 and 20.2 have the same output format, but the contents of the output and the behavior might be different, so make sure you use a 20.1 installation to generate 20.1 example output, and a 20.2 installation to generate 20.2 output.

The statement output formatting changed for CockroachDB in 19.2, and then again in 20.1, but, as you can see, we haven't made time to update all of our example output (hence the different formats here). I know it's not a part of #7671, but it would be helpful if you updated the output of all of the existing examples on the page to match the latest minor version for both 20.1 and 20.2 pages! :)

**Comment by ericharmeling** [comment]
> @rytaft and @Amruta-Ranade, I’m planning to add the following line to explain `<@` and `@>`. Please let me know if this works.
> 
> You can use the following operators to run containment queries on array columns:
> 
> * "is contained by": `<@`
> * "contains": `@>`

Great idea! I suggest changing this a bit to read:

~~~
You can use the [operators](functions-and-operators.html#supported-operations) `<@` ("is contained by") and `@>` ("contains") to run containment queries on `ARRAY` columns.
~~~

File: v20.1/array.md [hunk 1]
```diff
@@ -119,6 +119,34 @@ Arrays in CockroachDB are 1-indexed.
...
 
+### Accessing an array column using containment queries
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT * FROM c WHERE ARRAY[42,99] <@ d;
+~~~
+
+~~~
++------------+
```
[the remark this answers is not in the document]
Comment (deeptikorwar) [reply v20.1/array.md]: Fixed.

**Comment by deeptikorwar** [comment]
@rytaft and @ericharmeling, thanks for your review and inputs. 

I've fixed the examples and output format. Please take a look and let me know if any further changes are required.

**Review by rytaft — APPROVED** [review]
:lgtm:  thanks!

> Reviewed 2 of 2 files at r1, 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8697#-:-MKKod3t5sTVuLbtAART:btapcqv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @awoods187 and @ericharmeling)

**Review by ericharmeling — APPROVED** [review]
LGTM!

**Comment by SWDevAngel** [comment]
@deeptikorwar LAST DAY TO ORDER!!!
Hi Deeptikorwar. Thank you for contributing to CockroachDB this year. As a token of our appreciation we would like to send you a gift. We tried to reach you via email, but if you didn’t see the message please DM me on our [community Slack](https://cockroa.ch/slack) `@lisa` so I can send you a link to order your gift. (If you don’t want a gift, we also have a charitable contribution choice.) All orders must be in by December 13 (that’s tomorrow!), so please send this asap. :)

**Comment by deeptikorwar** [comment]
@SWDevAngel Thank you very much for the message and the gift. I used the link in my email a couple of days ago and went for the charitable contribution option this time :)

**Pull request merged**